lua/trunk.lua

```diff
@@ -40,6 +40,37 @@ local function findConfig()
 	return configDir and configDir .. "/.trunk/trunk.yaml" or ".trunk/trunk.yaml"
 end
 
+local function trim(s)
+	return s:match("^%s*(.-)%s*$")
+end
```

Comment:
At some point we may want to move all these string helper functions to their own file